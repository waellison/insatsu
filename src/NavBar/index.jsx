const NavBar = () => {
    return (
        <nav>
            <menu>
                <li>
                    <a href={"/"}>Home</a>
                </li>
                <li>
                    <a href={"/posts/all"}>All Posts</a>
                </li>
            </menu>
        </nav>
    )
}

export default NavBar