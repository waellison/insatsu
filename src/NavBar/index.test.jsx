import React from 'react';
import renderer from 'react-test-renderer';
import NavBar from './index';

test('NavBar renders as expected', () => {
    const navBar = renderer.create(<NavBar/>);
    let tree = navBar.toJSON();
    expect(tree).toMatchSnapshot();
});
