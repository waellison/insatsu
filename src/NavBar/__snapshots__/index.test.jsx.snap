// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar renders as expected 1`] = `
<nav>
  <menu>
    <li>
      <a
        href="/"
      >
        Home
      </a>
    </li>
    <li>
      <a
        href="/posts/all"
      >
        All Posts
      </a>
    </li>
  </menu>
</nav>
`;
